### PR TITLE
Add example: textured quad

### PR DESCRIPTION
**Pull request by dasifefe**
This is meant to be the simplest example to render a quad with texture, no light/shadow, and with orthographic camera.

The current issue that needs to be solved is that the image rendered is darker than the original.

image
The texture on top has the correct colors, how it should be.

## Checklist

- Cargo clippy/deny reported no issues.
- Everything ran normally on my end.
- No credits needed.

### CODE REVIEW

File: examples/textured-quad/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Renderer core
+rend3 = { version = "^0.2.0", path = "../../rend3" }
+# Programmable render list that dictates how the scene renders
+rend3-pbr = { version = "^0.2.0", path = "../../rend3-pbr" }
+# Provides `block_on` to wait for futures from sync code
+pollster = "0.2"
+# windowing
+winit = "0.25"
+# Load image
+image = "0.23.14"
```
Comment (cwfitzgerald) [inline examples/textured-quad/Cargo.toml]: Could we get default features off and only `png` turned on here?

File: examples/textured-quad/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        format: rend3::types::TextureFormat::Rgba8UnormSrgb,
+        size: glam::UVec2::new(image_checker.dimensions().0, image_checker.dimensions().1),
+        mip_count: rend3::types::MipmapCount::ONE,
+        mip_source: rend3::types::MipmapSource::Uploaded,
+    };
+    let texture_checker_handle = renderer.add_texture_2d(texture_checker);
+
+    // Add PBR material with all defaults except a single color.
+    let material = rend3_pbr::material::PbrMaterial {
+        albedo: rend3_pbr::material::AlbedoComponent::TextureValue {
```
Comment (cwfitzgerald) [inline examples/textured-quad/src/main.rs]: This could just be `Texture`, not `TextureValue`

**Review by cwfitzgerald — APPROVED** [review]
Some small changes, but after that we're good!

File: examples/textured-quad/Cargo.toml (hunk 1 above)
Comment (cwfitzgerald) [reply examples/textured-quad/Cargo.toml]: Small nit: we only actually need to require `0.23` not `0.23.14`

**Comment by dasifefe** [comment]
I will make the changes right now.

**Review by cwfitzgerald — APPROVED** [review]
Thanks so much!

**Pull request merged**